InterventionSkillExample/main.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from fastapi import FastAPI
+
+app = FastAPI()
+
+model = None
+
+@app.post('/invoke')
+def run(params: dict):
+
+    try:
+        payload = params["payload"]
+        profile = payload["profiles"]
+        intervention = payload["intervention_id"]
+        message = ""
+        try:
+            message = "Successfully invoked action for given profile ID "+str(profile)+"for intervention "+str(intervention)
+        except Exception as e:
+            print("Error in invoking action")
+
+        print(datetime.datetime.now())
```

Comment:
What is this useful for ? Consider removing this if it is for purely debugging purposes